src/formpack/reporting/export.py

```diff
@@ -519,7 +523,10 @@ def format_line(line, sep, quote):
                         yield format_line(row, sep, quote)
 
     def to_geojson(
-        self, submissions: Iterator, flatten: bool = True
+        self,
+        submissions: Iterator,
+        flatten: bool = True,
+        geo_question_name: Union[str, None] = None,
```

Comment:
Prompted me to see if there's an easier way to express this. Arguably, there is:
> `Optional[X]` is equivalent to `Union[X, None]`.

https://docs.python.org/3/library/typing.html#typing.Optional